server/public/model/config_test.go

```diff
@@ -1588,3 +1588,77 @@ func TestConfigDefaultCallsPluginState(t *testing.T) {
 		assert.False(t, c1.PluginSettings.PluginStates["com.mattermost.calls"].Enable)
 	})
 }
+
+func TestConfigGetMessageRetentionHours(t *testing.T) {
```

Comment:
These tests can be written more compact by using the range pattern used in other tests, but that is not a blocker.

Reply:
Done